src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/WiresDockingControl.java

```diff
@@ -30,6 +30,8 @@
 
     Point2D getCandidateLocation();
 
+    Point2D getAdjustedCandidateLocation(WiresContainer parent);
```

Comment:
would like to keep interface as simple as possible... why this new method? who is the caller?

Reply:
mmm as far as I understand the goal for this is while dragging, to adjust to the right location, isn't it? Because if that's the case, I see this as a private method in the default implementation (`WiresDockingControlImpl`), makes sense?

Reply:
hey @romartin this specific method I created to be used when Docking a node directly from the palette, in this case you remember the docking control is not activated because we don't have the shape yet, so this method is used to get a Point adjusted to the border of the parent using the intersection of a projection on the edge. Anyway I agree that we should keep the interface simple, so this can be inserted on the already existing `getCandidateLocation()` just verifying if the intersection was already set (in case of docking a shape) or not (in case dock directly from the palette), in this way we can keep only one method for this.